app/javascript/flavours/glitch/reducers/local_settings.js

```diff
@@ -37,6 +37,7 @@ const initialState = ImmutableMap({
       reblogs          : false,
       replies          : false,
       media            : false,
+      height           : 500,
```

Comment:
```suggestion
      height           : 400,
```
This should also be '400' to match the current default.